### DISCUSSION TITLE
install/community/k8s/online/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1853] [Announcements]
# install/community/k8s/online/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/install/community/k8s/online/

**Comment by HenrrryLiu** [comment]
安装过程中遇到一个问题，mysql-operator和minio-operator 起不来

mcamel-system      minio-operator-7876595dcf-25zhx                            0/1     CrashLoopBackOff    7 (108s ago)    20m
mcamel-system      minio-operator-7876595dcf-hh7zc                            1/1     Running             7 (5m13s ago)   12m
mcamel-system      mysql-operator-0                                           1/2     Running             0               20m
mcamel-system      mysql-operator-1                                           0/2     ContainerCreating   0               4m22s
mcamel-system      mysql-operator-2                                           1/2     Running             0               20m

**Reply by HenrrryLiu** [reply]
啊搞定了

**Comment by wangyao980112** [comment]
mcamel-system   mcamel-common-es-cluster-masters-es-data-0                  0/1     Init:ImagePullBackOff   0               62m
mcamel-system   mcamel-common-kpanda-mysql-cluster-mysql-0                  0/4     Init:ImagePullBackOff   0               62m
mcamel-system   mcamel-common-minio-cluster-pool-0-0                        1/2     ErrImagePull            0               81m
mcamel-system   mcamel-common-mysql-cluster-mysql-0                         0/4     Init:ImagePullBackOff   0               62m
mcamel-system   mysql-mgr-operator-7c5cd87948-cwzpm                         0/1     ErrImagePull            0               20m
mcamel-system   mysql-operator-0                                            1/2     Running                 5 (44m ago)     86m
mcamel-system   rfr-mcamel-common-redis-cluster-0                           0/2     ImagePullBackOff        0               62m
mcamel-system   rfs-mcamel-common-redis-cluster-7548b458df-ccjxf            0/1     Init:ImagePullBackOff

在线的镜像拉不到，单独拉日志报429请求过多

**Reply by jzhupup** [reply]
麻烦提供下拉取不到的详细原因啊？看下pod 事件

**Comment by Zane-LL** [comment]
安装过程要下载的文件这个文件有问题吧

3:09PM: + curl --retry 10 --retry-max-time 60 -LO https://files.m.daocloud.io/github.com/istio/istio/releases/download//istio--linux-amd64.tar.gz
3:09PM:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
3:09PM:                                  Dload  Upload   Total   Spent    Left  Speed
3:09PM: 100    19  100    19    0     0     24      0 --:--:-- --:--:-- --:--:--    24
3:09PM: + tar -zxf istio--linux-amd64.tar.gz
3:09PM: 
3:09PM: gzip: stdin: not in gzip format
3:09PM: tar: Child returned status 1
3:09PM: tar: Error is not recoverable: exiting now
3:09PM: + cleanup
3:09PM: cleanup
3:09PM: + echo cleanup
3:09PM: + set +ex
3:09PM: [sched-info] Sched shutdown...                                                                                                                                      
3:09PM: --------------------------------------
3:09PM: Unfortunately, the dce5 installation failed at step [8] ....
3:09PM: 
3:09PM: You can add -j8+ to your installation command to retry from last step.

**Comment by HirokiChin** [comment]
无权限拉取这个镜像，导致安装失败
Failed to pull image "docker.m.daocloud.io/ksmartdata/mysqld-exporter:v0.13.0-1": failed to pull and unpack image "docker.m.daocloud.io/ksmartdata/mysqld-exporter:v0.13.0-1": failed to resolve reference "docker.m.daocloud.io/ksmartdata/mysqld-exporter:v0.13.0-1": pulling from host docker.m.daocloud.io failed with status code [manifests v0.13.0-1]: 403 Forbidden

**Comment by paochiang** [comment]
安装报错了，有人能帮忙看下吗？
./dce5-installer install-app -z
./dce5-installer: line 1: syntax error near unexpected token `<'
./dce5-installer: line 1: `<!DOCTYPE html><html><head><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1.0"><link rel="icon" href="/.safeline/static/favicon.png" type="image/png"><title id="slg-title"></title><style>:root{--primary-color:#0067B8;--light-primary-color:#0067B8cc;--font-color:#fff;--light-font-color:#ffffff80;--success-color:#00b87c;--warning-color:#ff6666;--warning-font-color:#fff;--warning-light-font-color:#ffffff80;}</style><style>html{height:100%}body{height:100%;margin:0;font-family:PingFang SC,Helvetica Neue,Helvetica,Arial,sans-serif}#slg-bg{background-color:var(--primary-color);z-index:100;width:100%;height:100%;position:fixed;inset:0}#slg-box{z-index:300;border-radius:.5rem;flex-direction:column;width:90%;max-width:40rem;height:15rem;padding:1rem 0;display:flex;position:fixed;top:50%;left:50%;transform:translate(-50%,-80%)}#slg-image{flex:3;align-items:center;width:100%;padding-top:1rem;display:flex}#slg-warning{margin-left:auto;margin-right:auto}#slg-caption{text-align:center;color:var(--font-color);flex:2}#slg-text{flex:1;font-size:1.5rem;line-height:4rem;display:inline}#slg-desc{color:var(--light-font-color);font-size:.8rem;line-height:2rem}#slg-copyright{text-align:center;z-index:2000;width:100%;height:10rem;font-size:1rem;position:absolute;bottom:0}#slg-more-info{color:var(--font-color);margin-bottom:1rem;font-size:.8rem;line-height:2rem}#slg-copyright a{color:var(--light-font-color);text-decoration:none}#slg-copyright a:hover,#slg-name{color:var(--font-color)}#slg-copyright-text{margin-top:1rem}</style><script></script><script>window.product_data = {"favicon":"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAIAAAACACAMAAAD04JH5AAAC/VBMVEUAAAASEhIDAwMvLy8LCwtGRkZCQkIODg4+Pj4GBgYZGRk7OzsxMTEICAgLCwtHR0c5OTkPDw8FBQVJSUkRERFISEgNDQ0MDAwMDAwxMTFAQEBDQ0M5OTkcHBwwMDAaGhpCQkIpKSkyMjJCQkIWFhYtLS0eHh4TExMbGxslJSUlJSVKSkolJSU/Pz82NjYqKiogICBLS0tKSkpJSUk5OTlMTEz///8UoJ0PxsITnZoQlJEVop8SnJkDAwMPko8Rl5REREQHBwcKCgo9PT0Vo6ASmpcOkY5GRkYRmZYNj4z8/f07Ozs/Pz8ODg4Tm5gREREWpaEOkI1BQUEUFBQQlpMsLCxISEg6PDwfHx8bGxsYGRkjJCQxMjI2Nzc4ODhKSkozNDQvLy8WFhYpKSkNyMQmJiYMjYoNy8cAlpIhIiIdHR0nKCg3Pj4AmZY1NTUIAADq8/IuLi4FysYqn5w0QD8RnpsAkI3M4eAQp6P0+fkOxL+209I2Q0MOlJEAko8Ajoo2PTze6+sxRkYhn5wRjYoAioYIExISvboAnJkcmpcXmJUSkI0wQ0IPFhaexMMWp6Moo6Aan5w2VVROTk7E3t0co58QhIENendESkoOAAAql5UVt7Mqm5gOf30HDQ0VAADw9/aFtrQjop8An5wQiIb4/PsblZJFLi/b6ukSwb50tLIklZIOi4gSqqc+QkIuPT2ozcxuoqAonJkimpcPlpMUurcej4wqbGpBNjY1JSYVHh4eAADj7+6RwcAvgH7I397A29pBsq8QV1VEPT1CIiQ3Fxkkq6gupKEuop+w0tEDzckUs69nsa9bp6UldnQPYF8rT04JNTMsISEmFxgxkI0giofU5uVWrqsRbmwSaWcsSkkiSkkfPz41Li5CKiy619aJv72XxcQfsa46qKUnX11AVlUuDhAdBwmeyMdMpKIAhoMZfXuBu7lIqqc+n5w5dHI8Z2U7TUwkEBF2urgDpKEehYIPSEcKPTwkNjYQKikJISArAwUJAwNalpMTOjntEwKJAAAANnRSTlMABPwO+vxHKPqM/Pz81rCLUkvkyEQnzMRVJ86wiRjk5NrRr6Kijo2IbPXv6+Xix8DA5eTg2mzStdHlAAATcUlEQVR42rSRMW7jMBBFVzColgYINlETwKWrJAigkt30LqZloxM4XS6wB3BDxAcgfIAAqtxQcaFatQsDqq3cYIdarWkLzgZKlAdCAvVn5s+Mfg0iitrXRMaPfCbE3Xyl9Wp+J8SMP8ZyEmJGJ1SWUy7u9apFd3S3e8Gn8sd6iPzkMRfaWwVQB1pJ8Hjiw8cfPopnc73qHLGHxn8ChcxnceRzRh1e8gcqTTD8D6S2m9APXPrE0aaPE/SzIzL8FIbM7wGT2OeOYy+0BoYADPqgP1dABlqLEVro7BnZXwNVXSu4LjHWtfA9e5kg2cMHGFtV1nykUts6kVTm6/4RB7LvZlSg2gPqdDH2cLAm3IMO7YW2AJzKfHn7t1ox9RdwoPoYe8yyI3UQ6AczpW/DfxjozwFYMGty0/d35jXLXunVF/ImfGIAnMoN95c3qAJgq+2bgUsbdVim6fKgLlsD87at7Hko3sjhHUyVHz/g6k21zw2cXFzRrMmfOlg3hTsT8n21qZ06g4GaDl8/JS664x+mqMpds7dUmSRni/rlOUtbls8vdUECQcK+2ZVVYRaLkE8M+g0UmAAlXkB11mW629bOWpu/H582ZXqi/P10fM9JcPV2l5Zr33QvHZLQwaf+fwgnY9bUoTAMc7lQAi5da1voH+jWIZ0qZOgQKZeAZBGxEMgSCrEOImeQLFmE0oJL8wtEmgwl5mbJZB0UxMVF0EH8A/6D+343sUclqQ+KnHO+8z2v5yS/8/eCckgBj7xYq3Wd4XC4smsjTdxBG9XsFRacbq0m4sWA/wDhPv+bJzjivypwP6eAS+iKWo2A/YB4QdPELi6gkLJbKFzFCY76L68LQjEFxV0+jw6sTXAwNXpeukrafqFwfYn2x/9/uh+4xeU7/juXc3bO4X2puKnbFSTAGRw/fwX+DNxB5OOcib0EZAa4BT8aQJ+BoBy7BSzmFUHKBAncaGh3CRv4fvy1k6lh5MIvZSIo+WPvwjn8P+FKg+Xm7Ru2WjE+2iwHKAA/JDj/+QAu4D+C68p/QUVWVbYIRDFYMFWVKzQnu1n6Ik9wAU2m/7RYlGRCkvDrqpKED5AxoplkEddpMMYsq+1omtO2LAwMTKIkIa5NpKqbNMQc9p5ClOE/y8k5mSDbhj0xNYb2ciTMGAb5yx1dFPVOlRIYRmohgUYbeRsOirPMM8hLiR+7ZNZ7C5kB1BiZPgTp63U29zzyUwLPm7N6nSoPMQAL33qMtiYJpHzmA1CEfxuAhZEffLUY+hK7LeHHsZvrMfyEPl6buI76fmErrmStr8CPQpYEALkiHoM0/8nOAULUenL0nhXWLfw56t0i8IsRbOa834Q/Rm/25yYyYYnqeCGz6qHV052nlgE/5yQ1wc3/B6ACEAA9+o2m7s/mYRWtESKhDXvZnM586DkonJplWvsuZBhUwzkKm3/6aKeid9w/J9/8SrsAKUfybYDFrPHwIWq6/Rm0S2bV89qE55VLj9PJq61r4h4ofJ1MH0vl78KqWWoHn1T48dCYLRCgsiUn8Uvgb0BlFyOMGre3d2Otqema05kF6yl6etZ0Pem/2BrXczBpv/QncWF7ug5mHUfT0WB8d/vaiEJjT3DwJmD0j7PyDWkijOP4oJfthW96EURQIVREvdu1neibE2qB3Iv27hYbGR7Iict0k6AUJ14btFPaemGB7R9mDQxqIsy0F6aiicxSsETDGP3zTSoEEfS92+7uuWst6uvt7p57fvf7fu73/PHohf16d9f4WICi7PWOwUYldVPjw77B+fn5wT4Ake5mBhiWAh+WAhsHHQ12igqMjXdp6TEIR2FpnIEXLrk1XVofYFgaAHWOwSY1eWMTMiJlZZkDmwYddfU1FMXSA+ukg3Ee4v70JXRraklHaBkg5AjPI9X/q3E+7AjV2ymajqQ9bgLg0mmYknswad81/TzDMgBoCNXVPg6c/V8EvBh4XHsx1AAAls08n+4iEYw78mkZoKUo9/hAJMKCQBmD2qQDG/5/2WP4k7XKCNAMy0YiA+OwKAoApw0zAO6a3OvpDFsEaDjncISTsZpnZ5v+jQHz8eyzmlgyXFt3DgUAAEqQXtdNgIBZoC8Bt9ujyj09lmEjKkHIURsOxmJPM33ISeqqSWQfaPsyT2OxYLjWESr5I2VmbFq3aXFjIWh7ANpaB9fVH0E0RCsEFx3hYDB2P3a+ngmoiuzubr0w6OVuIKJ1M/Xn8UIwGHYUJwDNQkja38VpPrjBXqBugu4WogCLKECxBAoB5kEYCMnYfeiurC937z7/+cSkn3iKA5LjYknYhzH+sn+pACjBoqEEh7QSHPZYOQ+HA7/1ayiALJJAQQg+hpLJN8nPr588KKNXvZ/fvEkm5aigYk/4K0IJrq3DBC4ezmP1HFYLUNVi5VS1oACMBqAQKAiYCgpENvs5PHK7vCZe9fZms4o5Bv+ibK/46/nkEmhWVo+yEvE75tEBvJ5bKICBgLLXAKEOEI70o5sdI6Mb1/+gjduvejsepRFYVwf7GjtF+kORWx5OB2g5BnNo32H9oQdLgCH8QaAgyAznQvf4joXrfv+kv6TJSf/KyiTZ9I9+usnfC52T3WGv+BMEDBYCgXB4X3ET4KxeVVw+G2AQWPwxOgIY2sQbq9e7c8PD3SUNd+dW3m/kug1Phkdnb9xqK7kr9kQ+JpDNc5qZlcNWgL9qjwbAjS/hHbNoiJJ6+Jm51qFWXWjMbfPi5jAekk9zHxN8PK7am3MtjXMagKdaGYMjnA5QWMyUJYhLQuJjq7NT11Aq5V8QElJUiM8NOTuH9J6UMzchijPxsv6ZxYIOwB2xQAesXp8PB35ebz4bKfNedCYhbuZstpRTVcpmy22KohSlaYkSViedthQ6VdlswztCYiZaBiCSzXt1P+sBAFThQUne/BrN0L/7J4SFFZsNeXUNjSZEKc7QiAeesNNt6If8qygPYxbC1/K6n7cKANVer6skrjAWUIYOJ1xKp2hia8pmzO+c2hKiElOKBYIovm8lQ5w4NrZEDIMhHxQYK3Cqn9dbDYAzXp/2oLAHAJPiiZ2U+fNXO+IzhiBJEnbnUiaEzgkQmBXYK2gf7POekXcBF6E0S/8uaaLVRND9VR5iYxAtbN+xQQSCXwCmSWya9MNOcFBv+fJrFF1GlJBQv44cYngSUqbCyDDCdIApQSqTbS3v0z0PWqoIgMIHlioHIEX5Vb+pvqm5XUGKU2RYdFYU9WLhcqccAPuhQABUWY67XJdL8v1YDCgZTRRozyaErzkTQuuEIM5GKSKekreL0U6bGcCAGVj84VMdXa7jlmoSYC9AlRe+ThCR2oiwssAnJGOc1M9vTWoAPHrNYvd+uHSAastRl95czjLUHyXF+a07ii+5HGfkXdegGWG1IgCTXSYAjlrOFFvNyjlNUxU0m+DJWabcDX0UBYwDobg44qwEQKdVv+bmy64zliO4kYVW/nsDVUnt/bOC8H7IhNC9w4vxdgJAGKlQAajhe16zvHzEcgpXFeAdACojxON8fEP2JxEmt4UePaZf3FZWggLQVg7gXb5Z0ynLSR1g+a2d+qva2nnjPwYnbjeNBFEw/hnA/nb5smZ60nKi+UpJzSpAZWEc+E1ytdtaR7cT/USZemTGigC65wkLzmrj2xJlhxAiX5EJlzLt9p42Xt0acXSO0rzY307Et8uME5grU3ybvf33fEvfAKDqF6PmE9o0FMfxgofBDkJBqJ6GMOofPOzkaAnmFAge4qE9tOmtICSFnEIiUkyQNLB0IAy0h1II4vTgLtLu1E4cvajg2EHQDQsqHubc9KAgehF/r2ney3tJYN+FJK8vL99P3v+XjAL4YBROJENfs2Zd4+PmS6uAbEJhxgfNq7csPSH18gcK4NLNWg1tN2s3oQhYp7Twmq4px1vQD3y3VvQ1IxZvGLph/vqsOQnpoQjADAzR/lLmci0UAEiUI3ZL+hGeceXrG1PTHd2hbQijlpTSkBBAqMuZKzWstxMhEWDFSUZYs5C9pmiKnojgOEYSgDB5SzyvZM6TQOVP2+DwhVyw5wrcF0XTwzAVD/Yr1pftrU9AssbGp4Q5o/2nQjzPZ5ZqNRVrdZlDjpR0a/uWYzkO/h3HF3TOevUaDX7v3puczjHxKeHlVeJXqy1lFglA7WB9WeJADMB96PEVBZ6RluNYj56hjhk1x6Zh6Q53AknL6wc1ArCYyUcARkcpADDPfgr1jfZXlK+oP8KzA5oxHeBoFAHIZy5C2c9UG+2lAKCH3H5lSZT/cXRYQmsB1CZOALA3ChxVcFQvZrKqSgDGBnhIksTBAXZoCwCmI+9nzZNwvKdBV0QLGM2BI9Hp2ftJxngGgBjUbGaugqX+nIhSTH0TAIJnfKN4Epan2AmzA0Xpe4KULkESJz9V4jmXObWAjo0g2GgVhKkk9CdNT0IAcEAAJN7rKUp8dgB1pS+F6bFIuNBqEL/KAizQlyA4kzo6MkSBFg0woKIGA9NpxmYHh7bXF9IkGkcjdWYHFEuwMspHAXbTAUBPXfbWfc3eeccuiJ5pZt8T0wB2MQDkQB4AzjYqmOhgwgsiSgp7MdgRgNmtB5F4QYRyMD+xy8KtYxO9JgkuYu7HTw6IX+UsAJyDcDUQEKwXeJFWjwCg1m5ZPS8aDQimFpuw39qxxb4YF19YB/+ZHRCcy4ByjfnqTOpo30gGIK39qY2eLqq+Zx++iC2cVqAcEgD2R2oVA+Sm74jyBKABZSCyAHYAgHX/0Obpp2v3oEnehSh64WQrcQIogUboNt/IT98RnaligGpl1OF4Wj17O/Z0oj0YUBehqvD8NoNwd0fxeFpcZ1QJvcD2TPBvigtwigHGApOmrfz4eJ1dFn62zJ63GaFs2z+aj2n/69uHSpunJYwBAPujXgCVwWJ1XsaqtsQYgbuDe13S4dhWL7z9YGAreE1K6qEb8xdbVbkaOs1XF2dvSrMRgKG/z5VZgp6Fe13aoBdkv5nUEjEgUZnb94cyAciGr8sXZKLS31WeJdj0uqb5nPV43BxAVQA6Nz4m3LPsrrfJ+vOrf0sy0QL+XnBBxllQhywQiuhqsiGEMJdZH3PThSYIosj6rjeg0k8PRQEyoI4zQL6AvxfM1etyfSa5NGzx5XKZh7+peDiirdt247Vx66EXLFGYsunyZTY9bK1hCfvAyRz5ZHJanq+HGvq7QrEcFyBYCbUxPhy6Zre9mZC+KOxCBoSal0+H/qgrQECYze+IU4Ii5ghO7jzpmkx5x9unaXefgH08fVHs+FGXOuoEsHL1eimU7P8upggQbKY20j1U293YuJOW+rcvY5N6PUd9OM0CANbQ3xNTETY2XFwbGb04dO90i2n+4p4/LEUAsvSn01ypHon1O3wxVd0iVRvJKOWa2D4uvuNHHUq5wJ/UglJEsv/vWqo/eHStb1Ab2aEHCh/s03TtHyoAIlwDcEOIZMGNoT9ms4CtCi4a/q7jSZDCFD4rfuwPb0QyADcBDDD3v3er120UCIOSg4QoQAJZhCbdFXmGk4IFBRIFslDe4Aq3212HrgodrREFThOlsUtXqU68AKKzXyFFylixXNysN2GBEIJ/dBMUdsO3M8O3C5Gt/cbj2zEDGnQZmN0W7v7sVyP18Pfl+ec86Lp9ugBAXPLfitxA+TrEnfOQ6D4eUQfMRcu5XI34uvKZT35rvDmK76MK+69b9hJsWJDhrETkJenIMTsxN59f8NH4DpPfGeeM0sSLqgmQW7fxDMdVREk0XTpmN7AUft/NId8FfzkFWY182L6RSKqmwIuSp+nSN7vxOH+cm9/qPyWRV02ABLFW6NDF4X04iNIPB04zq/zsdFxHw1+mEdP3KDlt6Fyx+STQII4oSeKl45j44UC7fx+NZZxAnwMS4tfb6QxP8KoO7GQdmL5zNHwzWCd2VR8CRos+fxZrDpCvJHMDt2EB3X59P3AzLL+6Pn0CO6DUHWAaijTwXf9wYFCQFkh/XV/5blO3vndgexQ2DiyENe7EPUwbhxv46/3025xP8HSIdDu4kD3BriL0kk0aOG4TfnffCdJN4oU1KsGWe2wsvtJsOPBsgP32IkKyB1joD8g/ZIREGM+ZQKtdUf1eDmoIQxLlsHCAfB6RsHb7XL+HAxnBDQs7ss2njulOJkwDJw7ep9cdZ5pvyQ7yzfxf9d5eryO8aYGQWRZPTGfSCcecxNmMEMg39XXMf+8CB8X+hFloEVLk8QLPt9umTV8XizgvCLHCmWU3oRxWYiHZAFhqJ6wFeMhW6QJaPmy4e2GcaX+RrjKoY+7ZCIsPRD4lrt/PgWFbgvUJs3BGgOI1X8Up9ixBn+5uSuNV/loQAAFWEyCyjd76/D+TbMF8mwfMBcVu+1YUm01RvG13BEDm29Qpiyxy+UOmgXoHWk1Ai5R4/0MrQCEdW2o1lOHf+hL27B1fxwxwSR6C7NhiNwkcA8Z1g4M1+vfpWAk0J5T7icoNaG6OAh2niCdXHKo6t2DRo2x29+kYXQXF6SWfBiwclgYWrRtnqzodKpS0l4cyUBly+RNBWURJY9T91DVJpAPPW/is/tCYQrsLfkX7oZ638BlgdBeqJA9KLabW6A1kSb1g8edFySka0qXWngHtUjLE/1CAj0yI6rWkXMqaIAwGgqDJl4p0rYpHlf//AwRjLRBEezc3AAAAAElFTkSuQmCC","name":"雷池","name_en":"SafeLine","product":"长亭","community":"社区版","link":"https://waf-ce.chaitin.cn/","link_en":"https://waf.chaitin.com/","self":true};</script></head><body> <div id="slg-bg"></div> <div id="slg-box"> <div id="slg-image"> <svg id="slg-warning" width="68" height="59"><g fill="var(--font-color)"><g><path d="M29.455 2.852c2.062-3.527 6.151-4.07 8.48 0 1.538 2.527 7.818 13.159 14.15 23.904l.827 1.401.412.7.823 1.396A32540 32540 0 0 1 67.03 52.144l.02.038c.26.507 2.626 5.356-1.267 6.818H3.356s-6.846-1.44-.983-9.723c2.345-3.963 8.37-14.306 14.423-24.7l1.008-1.73c4.476-7.689 8.855-15.211 11.651-19.995m4.526 40.47c-2.157 0-3.905 1.74-3.905 3.885s1.748 3.884 3.905 3.884 3.905-1.739 3.905-3.884-1.748-3.884-3.905-3.884m.042-23.955c-2.18 0-3.947 1.758-3.947 3.926V35.69c0 2.168 1.767 3.926 3.947 3.926s3.947-1.757 3.947-3.926V23.293c0-2.168-1.767-3.926-3.947-3.926"/></g></g></svg> </div> <div id="slg-caption"> <div id="slg-text"></div> <div id="slg-desc"></div> </div> </div> <div id="slg-copyright"> <div id="slg-more-info"></div> <a id="slg-link"> <div id="slg-logo" style="display:none"> <svg width="32" height="35"><g fill="var(--font-color)"><path d="M15.006.33c.602-.44 1.4-.44 2.002 0 1.985 1.444 6.911 4.473 12.901 4.631.78.035 1.418.599 1.577 1.356.922 4.754 2.605 20.848-15.452 28.35C-2.077 27.183-.43 11.07.528 6.317c.142-.757.815-1.32 1.577-1.356 5.99-.158 10.863-3.187 12.9-4.63m1.037 4.54c-.28 1.647-2.15 1.938-2.15 1.938-1.9.309-2.819-1.12-2.819-1.12.82 2.255 2.198 2.391 2.446 2.397h2.423c-.7 1.802-3.48 2.133-3.48 2.133-3.159.39-4.689-1.423-4.689-1.423q.17.357.358.66l-.008-.005a11 11 0 0 0-3.106 7.671c0 6.09 4.937 11.026 11.026 11.026 6.09 0 11.027-4.936 11.027-11.026a11 11 0 0 0-3.11-7.674q.185-.3.353-.652s-1.53 1.816-4.69 1.423c0 0-2.776-.33-3.478-2.132h2.42c.245-.006 1.627-.14 2.448-2.397 0 0-.92 1.428-2.82 1.12-.142-.025-1.882-.356-2.15-1.94"/><polygon points="15.98353 17.9879553 9.8818726 21.4510476 15.3313444 24.6578974 17.2903808 23.6211992 13.5799337 21.4510476 15.98353 20.0985396 20.3159976 22.5564681 20.3159976 23.3648458 22.2042418 24.5010295 22.2042418 21.4510476" transform="rotate(-180 16.043 21.323)"/><polygon points="15.9835296 10.9942305 9.8818722 14.4573228 15.331344 17.6641726 17.2903804 16.6274743 13.5799333 14.4573228 15.9835296 13.1048148 20.3159972 15.5627433 20.3159972 16.371121 22.2042414 17.5073047 22.2042414 14.4573228"/></g></svg> </div> <div id="slg-copyright-text"> <span id="slg-prefix"></span> <span id="slg-name"></span> <span id="slg-suffix"></span> </div> </a> </div> <script>const e={unknown:{en:"Unknown Error",zh:"未知错误"},title:{en:"Protected By "+window.product_data?.name_en+" WAF",zh:window.product_data?.name+" WAF"},prefix:{en:"Security Detection Powered By",zh:"安全检测能力由"},suffix:{en:"",zh:"驱动"},name:{en:window.product_data?.name_en+" WAF",zh:window.product_data?.name+" WAF"},link:{en:window.product_data?.link_en,zh:window.product_data?.link},decrypting:{en:"Dynamic Decrypting",zh:"网页被保护，正在解密中"},failed:{en:"Decryption Failed",zh:"解密失败"},blocking:{en:"Access Forbidden",zh:"访问已被拦截"},"attack-desc":{en:"Blocked For Attack Detected",zh:"请求存在恶意行为，已被管理员拦截"},"too-fast-desc":{en:"Blocked for Access Too Fast",zh:"请求频率过高，已被管理员拦截"},"page-not-found-desc":{en:"The Page You Visited Does Not Exist",zh:"您访问的页面不存在"},"site-not-found":{en:"Website Not Found",zh:"网站不存在"},"site-not-found-desc":{en:"The Domain Name You Visited Does not Match The Server",zh:"您访问的域名与服务器不匹配"},offline:{en:"Website is Offline, Please Visit Later",zh:"网站维护中，暂时无法访问"},"gateway-error-desc":{en:"Server Response Error, Please Try Again Later",zh:"网站服务器异常，请稍后再试"},"gateway-timeout-desc":{en:"Server Response Timeout, Please Try Again Later",zh:"网站服务器响应超时，请稍后再试"},"it-works":{en:"It Works!",zh:"网站搭建成功"}};function n(n){let t=e[n];for(language in void 0===t&&(t=e.unknown),t)if(navigator.language.startsWith(language))return t[language];return t.en}function t(e,n,t){let o=document.getElementById(e);o&&(o[n]=t)}t("slg_title","innerText",n("title")),t("slg-link","href",n("link")),t("slg-prefix","innerText",n("prefix")),t("slg-name","innerText",n("name")),t("slg-suffix","innerText",n("suffix")),window.product_data?.self?document.getElementById("slg-logo").style.display="block":(document.getElementById("slg-logo").remove(),document.querySelector('link[rel="icon"]').href=window.product_data?.favicon);const o=document.getElementById("slg-box");o&&(o.style.display="none");</script> </body></html><script>console.log("hashID: a8023d9");</script><script>function install(url, err) {    var tag = document.createElement("script");    tag.src = url;    tag.onload = function() {        SafeLineChallenge("bf400cac948c45e5bae3baa83cc74080_4", {            host: "challenge.rivers.chaitin.cn",            local: url.startsWith("/"),            level: "1"        });    };    tag.onerror = err;    document.head.appendChild(tag);}document.addEventListener("DOMContentLoaded", () => {install("https://challenge.rivers.chaitin.cn/challenge/v2/challenge.js", function () {    install("/.safeline/challenge/v2/challenge.js");});});</script>'
